lib/puma/client.rb

```diff
@@ -244,19 +244,17 @@ def eagerly_finish
     end # IS_JRUBY
 
     def finish(timeout)
-      return true if @ready
```

Comment:
Hm, what's the impact of changing the return value here? Any?

Reply:
Just looking through the rest of the code, seems like we never checked the value.